packages/react/src/AnchoredOverlay/AnchoredOverlay.features.stories.tsx

```diff
@@ -2,17 +2,49 @@ import React, {useEffect, useRef, useState} from 'react'
 import type {Args, Meta} from '@storybook/react'
 import {FocusKeys} from '@primer/behaviors'
 
-import {Box, Button} from '..'
+import {Avatar, Box, Text} from '..'
 import {AnchoredOverlay} from '../AnchoredOverlay'
 import Heading from '../Heading'
+import Octicon from '../Octicon'
+import {Button} from '../Button'
 import {registerPortalRoot} from '../Portal'
 import {Playground} from './AnchoredOverlay.stories'
+import {LocationIcon, RepoIcon} from '@primer/octicons-react'
+import {Stack} from '../Stack/Stack'
 
 export default {
   title: 'Components/AnchoredOverlay/Features',
   component: AnchoredOverlay,
 } as Meta
 
+const hoverCard = (
+  <Stack gap="condensed" style={{minWidth: '320px', padding: '16px'}}>
+    <Stack direction="horizontal" gap="condensed" justify="space-between">
+      <Avatar src="https://avatars.githubusercontent.com/u/92997159?v=4" size={48} />
+      <Button size="small">Follow</Button>
+    </Stack>
+    <Stack direction="horizontal" gap="none">
+      <Text weight="medium">monalisa</Text>
+      <Text color={'var(--fgColor-muted)'} ml={1}>
+        Monalisa Octocat
+      </Text>
+    </Stack>
+    <Text fontSize={14}>Former beach cat and champion swimmer. Now your friendly octapus with a normal face.</Text>
```

Comment:
I'd recommend using the `size` prop instead.
```suggestion
    <Text size="medium">Former beach cat and champion swimmer. Now your friendly octapus with a normal face.</Text>
```

Same goes for other places where you're using `Text` to set the size.

Reply:
ah perfect, will do! thankss 🙏🏼